nucypher-core-python/nucypher_core/__init__.pyi

```diff
@@ -431,6 +434,40 @@ class ThresholdDecryptionRequest:
         ...
 
 
+class E2EEThresholdDecryptionRequest:
+
+    def decryption_request(self) -> ThresholdDecryptionRequest:
```

Comment:
This and the method below are marked as `#[getter]` in the bindings, so they will be property-like in Python

Reply:
✔️